dev/com.ibm.websphere.appserver.features/visibility/public/messagingClient-3.0/io.openliberty.messagingClient-3.0.feature

```diff
@@ -2,7 +2,7 @@
 symbolicName=io.openliberty.messagingClient-3.0
 visibility=public
 singleton=true
-IBM-API-Package: jakarta.jms; version="3.0"; type="spec", \
+IBM-API-Package: jakarta.jms; type="spec", \
```

Comment:
This is correct to remove the version.  Did you intend to remove the other versions instead of update them?